openssh-ldap-authkeys.spec

```diff
@@ -1,9 +1,13 @@
 # Enable Python dependency generation for Fedora and EL8+
 %{?python_enable_dependency_generator}
 
+%if "%{_vendor}" != "debbuild"
+%global with_selinux 1
+%endif
+
 Name:		openssh-ldap-authkeys
 Version:	0.2.0
-Release:	0%{?dist}
+Release:	2%{?dist}
```

Comment:
Was this my fault? Oops?

Reply:
I incremented the release number because this PR in its entirety represents a packaging/distro-specific change.